package.json

```diff
@@ -40,7 +42,7 @@
     "react-final-form": "^6.3.3",
     "react-final-form-arrays": "^3.1.1",
     "sequelize": "5.21.2",
-    "sqlite3": "4.1.0"
+    "sqlite3": "^4.1.0"
```

Comment:
Don't you think it's better to have fixed version of this so vital libraries, to be consistent with node version and so?

Reply:
Yes, I do. Probably range mark has occurred when I've synchronized with master and I've used `npm install`. It will be fixed.